spec/graph_ql/query_spec.rb

```diff
@@ -50,6 +50,26 @@
       assert_equal(GraphQL::Nodes::FragmentDefinition, query.fragments['cheeseFields'].class)
     end
 
+    describe 'scalar coercion' do
```

Comment:
Moved to `scalar_type_spec.rb`, which I think makes sense (the scalars are of type `ScalarType`) and will remain small.